site/zenodo_rdm/curation/tasks.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2024 CERN.
+#
+# ZenodoRDM is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+"""Tasks for curation."""
+
+from datetime import datetime, timedelta
+
+from celery import shared_task
+from flask import current_app
+from invenio_access.permissions import system_identity
+from invenio_rdm_records.proxies import current_rdm_records_service as records_service
+from invenio_search.engine import dsl
+
+from zenodo_rdm.curation.curators import EURecordCurator
+
+
+@shared_task
+def run_eu_record_curation(since):
+    """Run EC Curator."""
+    ctx = {"processed": 0, "approved": 0, "failed": 0, "since": since}
+    dry_run = not current_app.config.get("CURATION_ENABLE_EU_CURATOR")
+    curator = EURecordCurator(dry=dry_run)
+
+    query = dsl.Q(
+        "bool",
+        must=[
+            dsl.Q("term", **{"metadata.funding.funder.id": "00k4n6c32"}),
+            dsl.Q("term", **{"is_deleted": False}),
+            dsl.Q(
+                "range",
+                created={
+                    "lte": (datetime.now() - timedelta(days=30)).isoformat(),
```

Comment:
```suggestion
                    "lte": (datetime.utcnow() - timedelta(days=30)).isoformat(),
```

or the "new way" using `datetime.timezone.utc`, since it's deprecated